### PR TITLE
Add recognition for IAE and 72 Months Insurance

### PR DESCRIPTION
**Pull request by sophie-la-li**

### CODE REVIEW

File: web_resources/FME_exporter.js [hunk 1]
```diff
@@ -236,16 +236,20 @@ $(function () {
 
             $pledge.find('.without-images .item .title').each((i, elBonus) => {
                 const bonus = $(elBonus).text().trim();
-                const insuranceRegexResult = /(\d+)\s+Month\s+Insurance/i.exec(bonus);
+                const insuranceRegexResult = /(\d+)(\s+|-)Month(s|)\s+Insurance/i.exec(bonus);
```
Comment (Ioni14) [inline web_resources/FME_exporter.js]:
```suggestion
                const insuranceRegexResult = /(\d+)(\s+|-)Months?\s+Insurance/i.exec(bonus);
```

File: web_resources/FME_exporter.js [hunk 2]
```diff
@@ -236,16 +236,20 @@ $(function () {
...
                     insurance = parseInt(insuranceRegexResult[1]);
                 }
                 const lti = /Lifetime\s+Insurance/i.test(bonus);
+                const iae = /IAE\s+Insurance/i.test(bonus);
 
                 if (lti) {
                     lifetimeInsurance = true;
                 }
+                if (iae) {
+                    insurance = 120;
```
Comment (Ioni14) [inline web_resources/FME_exporter.js]: I would prefer directly to send IAE flag in the payload. ;)

I'll make changes FM side. :)

**Comment by Ioni14** [comment]
Thanks for contributing. :)
Really appreciated!

**Review by Ioni14 — CHANGES_REQUESTED** [review]

File: web_resources/FME_exporter.js (hunk 2 above)
Comment (sophie-la-li) [reply web_resources/FME_exporter.js]: Fine for me. Will change it. I wanted it to work with FM immediately yesterday ;-)

Comment (sophiekuehn) [reply web_resources/FME_exporter.js]: Question: Atm you can set insurance months AND lti AND iae, what is actually not possible. Wouldn't it be a good idea to, instead, introduce a insurance_type variable set to "lti" OR "iae" OR, lets say, "monthly"? I don't want to alter your code without asking in such a way ;-)

Comment (sophie-la-li) [reply web_resources/FME_exporter.js]: Question: Atm you can set insurance months AND lti AND iae, what is actually not possible. Wouldn't it be a good idea to, instead, introduce a insurance_type variable set to "lti" OR "iae" OR, lets say, "monthly"? (Plus insurance_months for the number) I don't want to alter your code too much without asking ;-)

Comment (Ioni14) [reply web_resources/FME_exporter.js]: Good idea, `insuranceType: enum<"lti"|"iae"|"monthly">` + `insuranceDuration: unsigned int`. :)

Comment (sophie-la-li) [reply web_resources/FME_exporter.js]: I made the changes. I would suggest to keep the properties Nullable for edge cases ;-)

**Pull request merged**